lib/app/timer.ex

```diff
@@ -1,15 +1,17 @@
 defmodule App.Timer do
   use Ecto.Schema
   import Ecto.Changeset
+  alias App.Item
```

Comment:
Does aliasing `App.Item` in the `timer.ex` 
and aliasing `App.Timer` in `item.ex`
create a circular reference? 
(I don't _think_ it does, but wondering ...)

Reply:
Hm, I don't believe using `alias` would create a circular reference (that's probably [why it's recommended instead of using `import`](https://stackoverflow.com/questions/70107659/in-elixir-why-is-alias-preferred-over-import-for-importing-the-modules). Regardless, this shouldn't be a problem or else everytime we wanted to reference another object in an Ecto schema as *belongs_to* or something similar, we'd have this problem 🤔